core/modules/system/css/system.css

```diff
@@ -370,3 +370,15 @@ section,
 summary {
   display: block;
 }
+
+.hide_comment_title {
+  display: block;
+  text-indent: -99999px;
+  line-height: 0;
+}
+.hide_comment_title:after {
+  display: block;
+  content: "Permalink";
```

Comment:
Caution, this won't be translatable.

Reply:
@indigoxela - Thanks. I suspected already that this is the wrong way to do this. I'm looking for ideas on the right way. Otherwise, I will experiment more later. Just wanted to get the concept working first. Suggestions welcome.